src/dataflow/render/delta_join.rs

```diff
@@ -337,18 +343,16 @@ where
     } else {
         let env = env.clone();
         let temp_storage = repr::RowArena::new();
-        updates.filter(move |input_row| {
+        let (ok_collection, err_collection) = updates.filter_fallible(move |input_row| {
             let datums = input_row.unpack();
-            ready_to_go.iter().all(|predicate| {
-                match predicate
-                    .eval(&datums, &env, &temp_storage)
-                    .unwrap_or(Datum::Null)
-                {
-                    Datum::True => true,
-                    Datum::False | Datum::Null => false,
-                    _ => unreachable!(),
+            for p in &ready_to_go {
+                if p.eval(&datums, &env, &temp_storage)? != Datum::True {
+                    return Ok(false);
                 }
-            })
-        })
+            }
+            Ok::<_, EvalError>(true)
+        });
+        err_collection.inspect(|e| println!("delta join filter err: {:?}", e));
```

Comment:
Should this be flagged with a TODO? It shouldn't be too hard to thread the errors back from here and in to the main line of error of delta join rendering.

Reply:
Whoops! Working on this now.

Reply:
This is fixed, but I'm stuck on another delta join thing now. (The thing I Slacked you about.)